core/src/com/unciv/models/tilesets/TileSet.kt

```diff
@@ -0,0 +1,26 @@
+package com.unciv.models.tilesets
+
+class TileSet(val name: String) {
+
+    var config = TileSetConfig()
+    var fallback: TileSet? = null
+
+    private val configs = HashMap<String, TileSetConfig>()
```

Comment:
What is the single config and what are these? Maybe more detailed names would help, currently I can't tell what either part is actually for

Reply:
"configs" map contains all configs gathered for this particular tileset (internal + mods), keys are mod-names or DEFAULT for built-in.

"config" is an clear config which serves as a place to merge those from above. This is the actual config we are interested in the game